documentation/asciidoc/computers/remote-access/remote-access-introduction.adoc

```diff
@@ -9,13 +9,91 @@ Any device connected to a Local Area Network is assigned an IP address. In order
 [[ip-address]]
 === How to Find your IP Address
 
-It is possible to find the IP address of your Raspberry Pi without connecting to a screen using one of the following methods:
+To find the IP of your Raspberry Pi, use one of the following methods.
 
-NOTE: If you are using a display with your Raspberry Pi and if you boot to the command line instead of the desktop, your IP address should be shown in the last few messages before the login prompt. Otherwise open a Terminal window and type `hostname -I` which will reveal your Raspberry Pi's IP address.
+==== Command Line
+
+Run the following command to output your IP address to the command line:
+
+----
+hostname -I
+----
+
+==== Boot Output
+
+If you use a display with your Raspberry Pi and you boot to the command line instead of the desktop, the boot sequence will include your IP address as one of the last few output messages before your login prompt.
 
 ==== Router devices list
 
-In a web browser navigate to your router's IP address e.g. `+http://192.168.1.1+`, which is usually printed on a label on your router; this will take you to a control panel. Then log in using your credentials, which is usually also printed on the router or sent to you in the accompanying paperwork. Browse to the list of connected devices or similar (all routers are different), and you should see some devices you recognise. Some devices are detected as PCs, tablets, phones, printers, etc. so you should recognise some and rule them out to figure out which is your Raspberry Pi. Also note the connection type; if your Raspberry Pi is connected with a wire there should be fewer devices to choose from.
+In a web browser, navigate to your router's IP address. This is often `http://192.168.1.1`, but you may be able to find it printed on a label on your router. This will take you to a control panel. Then log in using your credentials, which is usually also printed on the router or sent to you in the accompanying paperwork. Browse to the list of connected devices or similar (all routers are different), and you should see some devices you recognise. Some devices are detected as PCs, tablets, phones, printers, etc. so you should recognise some and rule them out to figure out which is your Raspberry Pi. Also note the connection type; if your Raspberry Pi is connected with a wire there should be fewer devices to choose from.
+
+==== Network Manager
+
+You can use the built-in Network Manager CLI (`nmcli`) to access details about your network. Run the following command:
```

Comment:
Does this (fairly verbose) section offer anything extra that the earlier (much simpler) `hostname -I` command doesn't? :shrug: 

Might also be worth mentioning that you can mouse-hover over the network-icon in the systray, and the tooltip will display your IP address.

Reply:
Good point. This section probably *could* be eliminated, but I also like the fact that if you have multiple active network interfaces -- say, you're plugged into ethernet *and* you're connected to wifi -- this will show you IPs for both. The latter sections are arguably even more esoteric, so... I would argue that if we want to eliminate this for redundancy, the same logic applies to the next few subsections.

FANTASTIC point about the mouse-hover. I did not know that, and honestly that should be the first suggestion in this whole section, since it doesn't require any CLI knowledge at all! Adding it now.